packages/cli/lib/axe-test-urls.js

```diff
@@ -3,15 +3,13 @@
 const WebDriver = require('selenium-webdriver');
 const AxeBuilder = require('@axe-core/webdriverjs');
 
-function testPages(urls, config, events) {
+async function testPages(urls, config, events) {
```

Comment:
Why is this an `async` function?

Reply:
I forgot to remove it when testing

Reply:
https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_ThenableWebDriver.html

Reply:
Michael and I paired on it. We tried to use the thenable interface as is without `await`ing it, and the unit tests for `startDriver` were causing us so many problems. We spent an hour trying to solve it and just gave up and made it work.